lib/adequate_crypto_address/cryptos.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+  class Cryptos
+    attr_reader :address, :raw_address
+
+    def initialize(address_string)
+      @address = address_string
+      # binding.pry
+    end
+
+    def valid?(address)
+      valid_prefix? ? true : false
+    end
+    # alias valid? valid_prefix?
+  end
```

Comment:
@snogrammer I commented this out bc it rendered an error ('valid_prefix? not found') so I'm prly not using the alias correctly- but I added the `address` argument to `valid?` bc I think it needs to refer to `valid?` method in `adequate_crypto_address.rb` 🧐